### ISSUE TITLE
Social Housing Provision Analysis using DL

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#439]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Social Housing Provision Analysis using DL</br>
:red_circle: **Aim** : The aim of this project is to analyze the housing dataset given here using DL methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/thedevastator/social-housing-provision-by-county-and-method</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Basma2423** [comment]
I Wanna give it a try, can you assign it to me? @abhisheks008 

Full name: **Basma Mahmoud**
GitHub Profile Link: [Basma2423](https://github.com/Basma2423)
Email ID: [mayarbasma2423@gmail.com](mailto:mayarbasma2423@gmail.com)
Approach for this Project: EDA + Multiple-architectures DL models + Accuracy Scores + Confusion Matrix,...
What is your participant role? (Mention the Open Source program): **GSSoC**-2024 participant

Can you add the label for **GSSoC**, please?
Thanks.

**Comment by gtanish2003** [comment]
Full name : Tanish Gupta
GitHub Profile Link :https://github.com/gtanish2003
Email ID : gtanish2003@gmail.com

Approach for this Project : Initially I would collect the dataset and then do the EDA to find the important features . 
                                         Then I start with ML algos like linear regression and random forest and then jump to deep learning 
                                         models like ANN.
What is your participant role? (Mention the Open Source program) GSSOC (Girlscript Summer of code

**Comment by abhisheks008** [comment]
Hi @gtanish2003 normal machine learning approaches will not work for this repo's projects. You need to implement deep learning methods for this project. Update your approach and post it here.

**Comment by DebanKsahu** [comment]
Full name :Deban Kumar Sahu
GitHub Profile Link :https://github.com/DebanKsahu
Email ID :sahudebankumar@gmail.com
Participant ID (if applicable):
Approach for this Project : I am new to deep learning as per my studies we can use a regression model for this as output is a continuous number. I will use sequential/functional api for this with layers like Dense and optimizers like Adam. After opening the csv i saw some NAN values and some unwanted features(if there any present) whom which i will drop using pandas. If i did not get satisfactory result then i will try to increase layers, and Rescale the values between 0-1 for better performance. If possible i will use any existing model by transfer learning for this problem.
What is your participant role? (GSSoC 2024)

**Comment by gtanish2003** [comment]
> Hi @gtanish2003 normal machine learning approaches will not work for this repo's projects. You need to implement deep learning methods for this project. Update your approach and post it here.

Sir I will start firstly aaply ann and then do hyperparameter tuning and try to use different loss function and different activations functions and different optimizers and if result is not satisfactory then I will apply rnn

**Comment by abhisheks008** [comment]
Both of you @gtanish2003 and @DebanKsahu can you please elaborate on the approaches and also be specific regarding the architectures of the models. Make sure you approach must have implementation of at least 2-3 deep learning methods for this dataset.

**Comment by gtanish2003** [comment]
I will use :
1. FNN (feedforward neural networks ) with use of dense and sequential layers 
2. LSTM

**Comment by DebanKsahu** [comment]
I am a beginner so I will use :
         1) Regression Model

**Comment by abhisheks008** [comment]
> I am a beginner so I will use : 1) Regression Model

You need to learn more about Deep learning. I really liked your enthisiasm on participating in this open source event. First go through the concepts of data analysis, then dive into machine learning concepts, make hands-on projects to test your understanding. After that once you confident about ML, try your hands dirty in Deep Learning. Diving directly will cause lapse in your understanding and concepts. See you soon in upcoming open source events as a deep learning pro! Good luck!

**Comment by abhisheks008** [comment]
> I will use :
> 
> 1. FNN (feedforward neural networks ) with use of dense and sequential layers
> 2. LSTM

Can you implement 2 more algorithms for this project/dataset?

**Comment by gtanish2003** [comment]
> > I will use :
> > 
> > 1. FNN (feedforward neural networks ) with use of dense and sequential layers
> > 2. LSTM
> 
> Can you implement 2 more algorithms for this project/dataset?

Further I can apply CNN and RNN

**Comment by abhisheks008** [comment]
Cool then issue assigned to you @gtanish2003
